hack/make-release-artifacts.sh

```diff
@@ -14,42 +16,42 @@
 
 # injects new image/tag into the images in ./release/kubernetes-manifests/demo.yaml 
 
-#!/usr/bin/env bash
 set -euo pipefail
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 log() { echo "$1" >&2; }
 fail() { log "$1"; exit 1; }
 
 TAG="${TAG?TAG env variable must be specified}"
 REPO_PREFIX="${REPO_PREFIX?REPO_PREFIX env variable must be specified}"
+out_file="${SCRIPTDIR}/../release/kubernetes-manifests/demo.yaml"
 
-# overwrite release/ with the latest manifests, adding "---" separator.
-src="./kubernetes-manifests/*"
-manifestfile="./release/kubernetes-manifests/demo.yaml"
-tmp="./release/kubernetes-manifests/tmp.yaml"
-[ -e $manifestfile ] && rm $manifestfile
-for f in $src; do (cat "${f}"; echo "---") >> $tmp; done
+read_manifests() {
+    local src_manifest_dir
+    src_manifest_dir="${SCRIPTDIR}/../kubernetes-manifests"
 
-# remove extra google headers 
-gsed -i '/^#/d' $tmp 
+    while IFS= read -d $'\0' -r file; do
+        cat "${file}"
+        echo "---"
+    done < <(find "${src_manifest_dir}" -name '*.yaml' -type f -print0)
+}
 
-# remove empty lines 
-gsed -r -i '/^\s*$/d' $tmp 
+# read and merge all manifests
+out_manifest="$(read_manifests)"
```

Comment:
trying to figure out how you're getting 1 google header to stay? 
also it's 2018 vs. 2019 now?

Reply:
I removed it because the license was there already, in the original files.